Makefile

```diff
@@ -8,15 +8,13 @@ BUILD_TMP = /tmp/syncstorage-build.${USER}
 PYPI = https://pypi.python.org/simple
 
 export MOZSVC_SQLURI = sqlite:///:memory:
-#export MOZSVC_SQLURI = mysql://user:password@localhost/my_sync_db
 
 # Hackety-hack around OSX system python bustage.
 # The need for this should go away with a future osx/xcode update.
 ARCHFLAGS = -Wno-error=unused-command-line-argument-hard-error-in-future
 
 INSTALL = ARCHFLAGS=$(ARCHFLAGS) $(PIP) install -U -i $(PYPI)
 
-
```

Comment:
These changes were noise in the original PR, restoring them to their original values.